numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -1027,6 +1027,8 @@ typedef void (NpyIter_GetMultiIndexFunc)(NpyIter *iter,
  * eliminate overlap.
  */
 #define NPY_ITER_COPY_IF_OVERLAP            0x00002000
+/* At least one operand uses UPDATEIFCOPY semantics, use context manager */
+#define NPY_ITER_USE_CONTEXT_MANAGER        0x00004000
```

Comment:
@seberg  "Why is there an explicit ResolveUpdateIfCopy in the nditer pywrap code? It seems to me that on the C-Side this should be triggered by the NpyIter_Deallocate call, ..."

Do you mean the call on line 2397 [here](https://github.com/numpy/numpy/pull/9269/files/6dc0c65e4b2ea67688d6b617da3a175cd603fc18#diff-7723df1c7a2da1cd528d452e6afbfd31R2397) in ``npyiter_exit``?